SCOPE.md

```diff
@@ -28,6 +28,12 @@ A `wbsearchentities` equivalent has not been included in the specification.
 
 Investigations will be carried out to determine if such functionality may make sense to be included as part of the primary entity retrieval / listing endpoint, or if search and lookup should remain a separate topic.
 
+### Merging Items & Redirects
+
+A `wbmergeitems` and `wbcreateredirect` equivalent have not been included in the specification.
+
+These would be included in the final spec.
```

Comment:
Will they be included in the final spec for sure or is it still up for debate?

Reply:
I believe that the direction that we want to go in will end up with these under the same rest endpoint, although I'm not sure exactly what they will look like right now.
An alternative would be to continue maintaining various used functionalities in the action API, but that would lead to a disjointed experience.
It certainly isn't one of the basic CRUD operations, but could end up being 2 Updates (so a slight differentiation from the current action way)?
There are probably quite a number of possibilities that we will have to look at at some point.

I could switch the wording to "probably"?

Reply:
I would say yes, I think there's still room for somehow mitigating this gap in functionality between the action APi and the rest API in a less obvious way than a 1:1 implementation.

Reply:
Looks like the concern let to an adjustment in wording. LGTM.